src/index.js

```diff
@@ -47,14 +47,14 @@ module.exports = {
 
     checkNextSiteHasBuilt({ publish, failBuild })
 
-    const { appDir, basePath, i18n, images, target, ignore } = await getNextConfig({ publish, failBuild })
+    const { appDir, basePath, i18n, images, target, ignore, outdir } = await getNextConfig({ publish, failBuild })
```

Comment:
Can you explain where you're getting this config name? I'm not aware of `outdir` being a standard config value.

Reply:
It's added when you're using Next.js with Nx. You can see the [source code adding it here](https://github.com/nrwl/nx/blob/67ff4acea20300de7ae316a957d310cd4fb40e5e/packages/next/src/utils/config.ts#L158-L168)